### PR TITLE
live: add ha with redis to what's new

### PR DESCRIPTION
**Pull request by FZambia**
**What this PR does / why we need it**:

Mentioning Live HA with Redis in What's new docs

### CODE REVIEW

File: docs/sources/live/live-ha-setup.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ ha_engine = redis
 ha_engine_address = 127.0.0.1:6379
 ```
 
+See a description of [ha_engine]({{< relref "../administration/configuration.md#ha_engine" >}}) and [ha_engine_address]({{< relref "../administration/configuration.md#ha_engine_address" >}}) options.
```
Comment (achatterjee-grafana) [inline docs/sources/live/live-ha-setup.md:32]:
```suggestion
For additional information, refer to the [ha_engine]({{< relref "../administration/configuration.md#ha_engine" >}}) and [ha_engine_address]({{< relref "../administration/configuration.md#ha_engine_address" >}}) options.
```

File: docs/sources/whatsnew/whats-new-in-v8-1.md [hunk 2]
```diff
@@ -127,6 +127,10 @@ Tests are most likely to be affected. There are some tips for fixing these in th
 
 We’d love as much feedback as possible about this change, because we are considering making this the default behavior in a future release of Grafana.
 
+### High availability setup support for Grafana Live
+
+We have added an experimental HA setup support for Grafana Live with Redis. This resolves several limitations of Grafana Live when clients were connected to different Grafana instances and instances had no shared state. For additional information, refer to [Configure Grafana Live HA setup]({{< relref "../live/live-ha-setup.md" >}}).
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-1.md:132]:
```suggestion
We have added an experimental HA setup support for Grafana Live with Redis. This resolves the limitation when clients were connected to different Grafana instances and those instances had no shared state. For additional information, refer to [Configure Grafana Live HA setup]({{< relref "../live/live-ha-setup.md" >}}).
```

**Review by achatterjee-grafana — COMMENTED** [review]
Minor comments. Thanks for updating the what's new topic!

**Review by achatterjee-grafana — APPROVED** [review]
lgtm

**Review by atifali — APPROVED** [review]
looks great!

**Pull request merged**